### PR TITLE
fix: show tooltips in chromatic tests

### LINKED ISSUE #1375: Automatically open tooltip for Chromatic tests
Chromatic is running visual regressions on the existing tooltip story, which is unopened when the snapshot is taken. 

I see two possible options: 
1. Create a stateless tooltip component (in addition to the existing stateful component). In the visual regression tests use the stateless components and default them to open. I can see use cases where a stateless variant would be useful for consumers. 
   1.a. Alternatively a "intiallyOpen" prop on the existing stateful component
2. Figure out a nice way to mock mouse interactions when running the story in Chromatic. 

Best explained with an image: 
Screen Shot 2021-04-06 at 2 34 19 pm

### PR DESCRIPTION
**Pull request by talbet**
# Objective

The visual regression tests around the tooltip component render the button that triggers the tooltip, but not the tooltip itself. This means that these tests provide little value, unless we can get the tooltip into the test.

The ideal way to do this would be to trigger a hover event in chromatic before the screenshot is captured, but the current chromatic API doesn't support any interaction or events. Instead, we are putting a new prop into the component to signal that the tooltip should be visible on the first render. This seems like it will only be useful for testing, but someone may find a valid use for it, so it hasn't been named obtusely (a name like `chromaticTestVisibility` would send a signal that this is the only valid usage)

# Motivation and Context

closes #1375 

# Screenshots (if appropriate)
Screen Shot 2021-07-30 at 9 36 02 am

# Checklist

- [ ] If this contains visual changes, has it been reviewed by a designer?
- [ ] If this introduces either a new component or a breaking change to an existing component, has it been reviewed by an advocate? (Ask in #prod_design_systems)
- [ ] I have considered likely risks of these changes and got someone else to QA as appropriate
- [ ] I have or will communicate these changes to the front end practice

# Additional Considerations
- Does there need to be right-to-left (RTL) options for localization and internationalization?
- Has the test suite been updated?
- Have you done cross-browser testing for our [supported browsers](https://academy.cultureamp.com/hc/en-us/articles/204539569-Supported-browsers-for-Participants)?
- Have you updated any relevant documentation or left useful comments in the code?
- Have you reviewed Culture Amp's Web Accessibility guide [Current Compliance, Going Forward, Statement and Answers for Customers](https://cultureamp.atlassian.net/wiki/spaces/Prod/pages/428572998/Web+Accessibility)?
- If this contains substantial visual changes, especially far reaching ones, have you unlocked the Chromatic step in the pipeline?
- Have Storybook stories been updated?

**If you're new to Kaizen, please ask #prod_design_systems to set up an onboarding session to get you up to speed.** If you have an urgent PR to merge before that happens, it is safest to ask in #prod_design_systems to have a design system advocate review the PR to catch any issues.

### CODE REVIEW

**Comment by talbet** [comment]
Here is a link to the chromatic build for this branch https://www.chromatic.com/test?appId=5e12c297b9d9020020789ca3&id=61033f5e581f7a0049ee812f

File: draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -36,6 +36,11 @@ export type TooltipProps = {
    * `overflow: scroll` or `overflow: hidden` element.
    */
   portalSelector?: string
+  /**
+   * Should the tooltip be visible on the first render. Useful for visual
+   * regression testing.
+   */
+  initiallyVisible?: boolean
```
Comment (ActuallyACat) [inline draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx:43]: /Nitpick It feels more consistent to call this `isInitiallyVisible`, as we have Modal (has `isInitiallyOpen` prop), and FilterMenuButton (has `isInitialDropdownVisible` prop) following this convention.

**Review by ActuallyACat — CHANGES_REQUESTED** [review]
Looks great - thanks @talbet and @aaronknol! I have a 🤏🏻 suggestion about the name of the new prop, but everything else LGTM.

File: draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx (hunk 1 above)
Comment (talbet) [reply draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx]: That makes sense. I've updated the variable name.

**Review by ActuallyACat — APPROVED** [review]

**Pull request merged**